pinot-broker/src/main/java/org/apache/pinot/broker/api/resources/PinotClientRequest.java

```diff
@@ -316,12 +316,22 @@ private BrokerResponse executeSqlQuery(ObjectNode sqlRequestJson, HttpRequesterI
         try (RequestScope requestStatistics = Tracing.getTracer().createRequestScope()) {
           return _requestHandler.handleRequest(sqlRequestJson, sqlNodeAndOptions, httpRequesterIdentity,
               requestStatistics, httpHeaders);
+        } catch (Exception e) {
+          LOGGER.error("Error handling DQL request:\n{}\nException: {}", sqlRequestJson,
+              QueryException.getTruncatedStackTrace(e));
+          throw e;
```

Comment:
@xiangfu0 any idea what kind of error we should wrap and rethrow here? simply pass through seems a bit less informative than I thought of but i couldn't think of a suitable one

Reply:
I would say, here should just be 500 internal error with some truncated stacktrace

Reply:
already merged. i think we can improve later (this PR only rethrow and doesn't change the behavior)